src/pages/LandingPage.vue

```diff
@@ -1,13 +1,91 @@
 <template>
-  <ProductList />
+  <div class="container">
+    <div class="navigation">
+      <v-button @click="showModal = !showModal">
+        Add Item
+      </v-button>
+    </div>
+    <Modal
+      v-if="showModal"
+      @close="showModal = !showModal"
+    >
+      <div slot="header">
+        <p>{{ action }}</p>
+      </div>
+      <template slot="body">
+        <add-edit-form 
```

Comment:
Nitpick: This is the reason why I was suggesting to have the `add-edit-form` as an actual `AddEditModal`. Then you wouldn't need to repeat yourself and here again wrap the component in a modal. 😃